src/js/global/pageManager.js

```diff
@@ -133,7 +133,18 @@ PageManager.prototype.change = function() {
     // Show current default page
     this.$page.style.display = 'block';
     this.$submitButton.style.display = 'block';
-    this.$sidebar.style.display = 'block';
+
+    if ( null == localStorage.getItem( 'wpr-show-sidebar' ) ) {
```

Comment:
Should be `===` to test against `null` (and while at it, you can also do the same for `'on'` and `'off'` 😛).